src/app/components/Headers/HeaderRecipes.tsx

```diff
@@ -0,0 +1,28 @@
+import { AppBar, InputBase, Paper } from '@mui/material';
+import React from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+
+export default () => {
+  return (
+    <AppBar className="header-flex">
+      <Paper
+        sx={{
+          display: 'flex',
+          width: '60%',
```

Comment:
This looks a little weird when I extend the page - maybe you can adjust the width.